src/overlays/actors/ovl_En_Blkobj/z_en_blkobj.c

```diff
@@ -39,12 +39,12 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneDownward, 300, ICHAIN_STOP),
 };
 
-static Gfx D_809C2590[] = {
+static Gfx sSetOpa[] = {
```

Comment:
For this kind of display lists, I suggest something a bit more descriptive like `sSetupOpaDL`